contracts/dao/voting/vesting-lp-vault/src/contract.rs

```diff
@@ -283,86 +275,50 @@ pub fn query_total_power_at_height(
 }
 
-pub fn query_dao(deps: Deps) -> StdResult<Binary> {
+pub fn query_dao(deps: Deps) -> ContractResult<Binary> {
     let dao = DAO.load(deps.storage)?;
-    to_binary(&dao)
+    Ok(to_binary(&dao)?)
 }
 
-pub fn query_name(deps: Deps) -> StdResult<Binary> {
+pub fn query_name(deps: Deps) -> ContractResult<Binary> {
     let config = CONFIG.load(deps.storage)?;
-    to_binary(&config.name)
+    Ok(to_binary(&config.name)?)
 }
 
-pub fn query_description(deps: Deps) -> StdResult<Binary> {
+pub fn query_description(deps: Deps) -> ContractResult<Binary> {
     let config = CONFIG.load(deps.storage)?;
-    to_binary(&config.description)
+    Ok(to_binary(&config.description)?)
 }
 
-pub fn query_config(deps: Deps) -> StdResult<Binary> {
+pub fn query_config(deps: Deps) -> ContractResult<Binary> {
     let config = CONFIG.load(deps.storage)?;
-    to_binary(&config)
+    Ok(to_binary(&config)?)
 }
 
 pub fn query_list_bonders(
-    deps: Deps,
-    start_after: Option<String>,
-    limit: Option<u32>,
-    order_by: Option<OrderBy>,
-) -> StdResult<Binary> {
-    // TODO: this method does not adjust LP tokens amount to their corresponding voting power
-    let config = CONFIG.load(deps.storage)?;
-
-    let query_msg = VestingLpQueryMsg::VestingAccounts {
-        start_after,
-        limit,
-        order_by,
-    };
-    let atom_vesting_accounts: VestingAccountsResponse = deps
-        .querier
-        .query_wasm_smart(config.atom_vesting_lp_contract, &query_msg)?;
-    let usdc_vesting_accounts: VestingAccountsResponse = deps
-        .querier
-        .query_wasm_smart(config.usdc_vesting_lp_contract, &query_msg)?;
-
-    let bonders = atom_vesting_accounts
-        .vesting_accounts
-        .into_iter()
-        .chain(usdc_vesting_accounts.vesting_accounts.into_iter())
-        .map(|vesting_account| BonderBalanceResponse {
-            address: vesting_account.address.into_string(),
-            balance: vesting_account
-                .info
-                .schedules
-                .into_iter()
-                .map(|vesting_schedule| vesting_schedule.start_point.amount)
-                .sum(), // TODO: probably deduct vesting_account.info.released_amount
-        })
-        .collect::<Vec<_>>();
-
-    to_binary(&ListBondersResponse { bonders })
+    _deps: Deps,
+    _start_after: Option<String>,
+    _limit: Option<u32>,
+) -> ContractResult<Binary> {
+    Err(ContractError::BondingDisabled {})
 }
 
 pub fn query_bonding_status(
     _deps: Deps,
-    env: Env,
-    height: Option<u64>,
+    _env: Env,
+    _height: Option<u64>,
     _address: String,
-) -> StdResult<BondingStatusResponse> {
-    let height = height.unwrap_or(env.block.height);
-    Ok(BondingStatusResponse {
-        unbondable_abount: Uint128::zero(),
-        bonding_enabled: false,
-        height,
-    })
+) -> ContractResult<BondingStatusResponse> {
+    Err(ContractError::BondingDisabled {})
```

Comment:
oh, I like it. a gentle way imo. could you please make the same for the lockdrop vault contract? so the contracts respond the same way. for both `query_list_bonders` and `query_bonding_status`